charts/jaeger/values.yaml

```
@@ -395,6 +395,7 @@ spark:
   tag: latest
   pullPolicy: Always
   cmdlineParams: {}
+  env: []
```

```suggestion
  extraEnv: []
```

opening this suggestion as per https://github.com/jaegertracing/helm-charts/issues/120#issuecomment-643047774 which is still open to discussion of course :)

---

@naseemkullah done